Lib/zipimport.py

```diff
@@ -20,6 +20,7 @@
 import _imp  # for check_hash_based_pycs
 import _io  # for open
 import marshal  # for loads
+import struct  # for ZIP64 handling
```

Comment:
hmm, sorry for backtracking on this, but I think we must defer this import!
with it at module scope, the `test_cmd_line_script.CmdLineTest.test_module_in_package_in_zipfile` test is failing, while with the deferred import it's passing.
maybe there an import cycle happening? I didn't dig deeper.

Reply:
I dug but got none the wiser. I did re-introduce the deferred import in the end.